src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
This allows us to mock (final) Kotlin classes.

Reply:
@matt-crowson-moj I switched to including `testImplementation("org.mockito:mockito-inline:3.4.6")`